lib/bitstyles_phoenix/component/button.ex

```diff
@@ -276,6 +492,23 @@ defmodule BitstylesPhoenix.Component.Button do
   defp variant_classes(variants) when is_list(variants),
     do: Enum.map(variants, &"a-button--#{&1}")
 
+  defp size_class(nil), do: ""
+
+  defp size_class(size) when is_binary(size) or is_atom(size),
+    do: "a-button--#{size}"
+
+  defp shape_classes(nil), do: []
+
+  defp shape_classes(shape) when is_atom(shape),
+    do: shape_classes(Atom.to_string(shape))
+
+  defp shape_classes(shape) do
+    case shape do
+      "round" -> ["a-button--square", "a-button--round"]
+      _ -> ["a-button--#{shape}"]
+    end
+  end
```

Comment:
Took a quick look at [the bitstyles docs](https://bitcrowd.github.io/bitstyles/?path=/docs/atoms-button--small-round-primary) and realized we were following a wrong track yesterday 🙈 Of course, for the full round button, if in the HTML we would apply both, the `square` and the `round` shape, we'll do exactly the same in the Phoenix helper as well. So we only need to differentiate between a list of shapes or a single shape as input (similar to how it's done for the variants):

```suggestion
  defp shape_classes(shape) when is_binary(shape) or is_atom(shape), do: shape_classes([shape])

  defp shape_classes(shapes), do: Enum.map(shapes, &"a-button--#{&1}")
```

So maybe we want to rename the variants helper and reuse it here.